bands/templates/band_admin/index.html

```diff
@@ -13,46 +13,52 @@
 }
 </style>
 
-<h1>Bands and Other Groups <a class="btn btn-primary" href="add_band">Add a Band</a></h1>
+<h1>Bands and Other Groups <a class="btn btn-primary" href="add_band">Add a Group</a></h1>
+  <div id="show-opts" class="btn-group" data-toggle="buttons"></div>
```

Comment:
Nitpick... maybe some padding between the button group and the "Show [All] entries" selector:
<img width="309" alt="screen shot 2017-09-13 at 10 57 46 pm" src="https://user-images.githubusercontent.com/2592431/30411028-8e6fb9da-98dc-11e7-90b5-fb4675a45bde.png">

Reply:
Oh yea, you know, I tried adding that but couldn't figure it out and then got distracted. Whoops. 😅